src/MarkdownTextInput.tsx

```diff
@@ -43,13 +39,33 @@ function initializeLiveMarkdownIfNeeded() {
   if (!global.jsi_setMarkdownRuntime) {
     throw new Error('[react-native-live-markdown] global.jsi_setMarkdownRuntime is not available');
   }
+
+  let createWorkletRuntime: (name: string) => WorkletRuntime;
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    createWorkletRuntime = require(workletsPackageName).createWorkletRuntime;
+  } catch {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    createWorkletRuntime = require('react-native-reanimated').createWorkletRuntime;
+  }
   workletRuntime = createWorkletRuntime('LiveMarkdownRuntime');
+
   global.jsi_setMarkdownRuntime(workletRuntime);
   initialized = true;
 }
 
-function registerParser(parser: (input: string) => MarkdownRange[]): number {
+function registerParser(parser: (input: string) => MarkdownRange[], workletsPackageName = 'react-native-worklets'): number {
```

Comment:
So the trick is to pass function argument (with a default value) to `require`, is that correct?

Reply:
Yes, that way metro can't resolve it in a build phase and only does that in runtime where error will be handled by `try/catch`